service/api_patch.go

```diff
@@ -155,11 +155,18 @@ func (as *APIServer) updatePatchModule(w http.ResponseWriter, r *http.Request) {
 		PatchString string `json:"patch"`
 		Githash     string `json:"githash"`
 		Message     string `json:"message"`
+		CommitQueue bool   `json:"commit_queue"`
 	}{}
 	if err = util.ReadJSONInto(util.NewRequestReader(r), &data); err != nil {
 		as.LoggedError(w, r, http.StatusBadRequest, err)
 		return
 	}
+
+	if p.Alias == evergreen.CommitQueueAlias && !data.CommitQueue {
```

Comment:
I think this will prevent a user from adding modules to commit queue patches until they update their CLI (since `data.CommitQueue` will be false)
Are we okay with this?

Reply:
I think so. I made the comment also mentioning updating the CLI.

Reply:
would it work to validate on the client side. We would know for sure what the user did wrong.

Reply:
It goes through the same code path on the client side, to it seemed simpler to do it this way.